pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -168,29 +219,41 @@ Function | Description
 
 For any function you defined as an `export` in your `gulpfile.js`, you can run `npx gulp [function]`.
 
-## Step 5: Initialize your project
```

Comment:
Reorganized this section to move the explanation of what the command does closer to the top.